github/Repository.py

```diff
@@ -968,13 +968,13 @@ def create_milestone(self, title, state=github.GithubObject.NotSet, description=
         :param title: string
         :param state: string
         :param description: string
-        :param due_on: date
+        :param due_on: datetime
         :rtype: :class:`github.Milestone.Milestone`
         """
         assert isinstance(title, (str, unicode)), title
         assert state is github.GithubObject.NotSet or isinstance(state, (str, unicode)), state
         assert description is github.GithubObject.NotSet or isinstance(description, (str, unicode)), description
-        assert due_on is github.GithubObject.NotSet or isinstance(due_on, datetime.date), due_on
+        assert due_on is github.GithubObject.NotSet or isinstance(due_on, datetime.date) or isinstance(due_on, datetime.datetime), due_on
```

Comment:
It would be cleaner to condense this:
```python
isinstance(due_on, datetime.date) or isinstance(due_on, datetime.datetime)
```
into this:
```python
isinstance(due_on, (datetime.date, datetime.datetime))
```

Reply:
Thanks for the review.  Changed syntax per your recommendation in commit 7c05c04ef6d903bf2f248a01f85333349b7d0486.